src/PluginInfo/PluginInfo.js

```diff
@@ -124,13 +124,28 @@ function PluginInfo (dirname) {
     // <config-file> tag
     self.getConfigFiles = getConfigFiles;
     function getConfigFiles (platform) {
-        var configFiles = _getTags(self._et, 'config-file', platform, _parseConfigFile);
+        var configFiles = _getTags(self._et, 'config-file', platform, _parseConfigFile.bind(this, platform));
         return configFiles;
     }
 
-    function _parseConfigFile (tag) {
+    function _parseConfigFile (platform, tag) {
+        var target = tag.attrib['target'];
+        if (platform == 'android') {
```

Comment:
We're currently trying to remove all platform specific code from cordova-common, so this might be a problem.